docs/contribute/style-guide.md

```diff
@@ -8,13 +8,11 @@ sidebar_position: 2
 Style guidelines help keep the Consensys documentation consistent, concise, and readable.
 Refer to the following guides when writing, editing, or reviewing doc content:
 
-- [**Microsoft Writing Style Guide**](https://learn.microsoft.com/en-us/style-guide/welcome/) -
-  Refer to this guide for style, voice, grammar, and text formatting guidelines.
-- [**Diátaxis framework**](https://diataxis.fr/) - Refer to this guide for information about
+- [**Microsoft Writing Style Guide**](https://learn.microsoft.com/en-us/style-guide/welcome/): Refer to this guide for style, voice, grammar, and text formatting guidelines.
+- [**Diátaxis framework**](https://diataxis.fr/): Refer to this guide for information about
   function-based docs.
-- [**Consensys Editorial Style Guide**](https://docs.google.com/document/d/1smRdw4TUIpz9re_o0_0DKdH_nK6cSPMJOK6BcbhjJ7Y/edit?usp=sharing)
-  - Refer to this guide for spelling and usage of blockchain-related terms.
-  This guide is only available to internal Consensys contributors.
+- [**Consensys Editorial Style Guide**](https://www.notion.so/consensys/Consensys-Editorial-Style-Guide-d5b9867e85df4ae38f8bed44f61a77d5): Refer to this guide for spelling and usage of blockchain-related terms. [Vale](run-vale.md) assists writers to adhere to this style.
+  > This guide is only available to internal Consensys contributors.
```

Comment:
Let's not use Markdown blockquotes as call-outs in Docusaurus docs. Use admonitions instead.
```suggestion
  :::note
  This guide is only available to internal Consensys contributors.
  :::
```